queue_services/entity-filer/src/entity_filer/filing_processors/admin_freeze.py

```diff
@@ -17,15 +17,13 @@
 
 import dpath
 
-# from entity_filer.exceptions import DefaultException, logger
+from entity_filer.exceptions import DefaultException, logger
```

Comment:
Any particular reason why this line should remain commented out in this file (and in the other filing_processors)? Most of these files make use of `DefaultException`, so commenting out this line causes an error.

Reply:
Thanks for catching this.  Not sure what happened there.